ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/RemoteImage.kt

```diff
@@ -80,17 +83,13 @@ private sealed class ImageSource {
 @Composable
 private fun Image(
     source: ImageSource,
+    placeholderSource: ImageSource.Remote?,
     modifier: Modifier = Modifier,
     contentScale: ContentScale,
     contentDescription: String?,
     transformation: Transformation?,
     alpha: Float,
 ) {
-    // Previews don't support images
-    if (isInPreviewMode()) {
```

Comment:
I noticed we could have some kind of previews by using `placeholder`. I think it should be safe as long as it's `inPreviewMode`, and more useful than the current solution.